assetto-corsa-comp.kvp

```diff
@@ -0,0 +1,81 @@
+Meta.Author=Greelan
+Meta.URL=https://steamdb.info/app/805550
+Meta.DisplayImageSource=steam:805550
+Meta.EndpointURIFormat=
+Meta.ConfigManifest=assetto-corsa-compconfig.json
+Meta.MetaConfigManifest=assetto-corsa-compmetaconfig.json
+Meta.ConfigRoot=assetto-corsa-comp.kvp
+Meta.MinAMPVersion=2.4.0.2
+Meta.SpecificDockerImage=cubecoders/ampbase:wine
+Meta.DockerRequired=False
+Meta.ContainerPolicy=Recommended
+Meta.Prerequsites=[]
+Meta.ConfigReleaseState=NotSpecified
+App.DisplayName=Assetto Corsa Competizione
+App.RootDir=./assetto-corsa-comp/
+App.BaseDirectory=./assetto-corsa-comp/1430110
+App.ExecutableWin=1430110\server\accServer.exe
+App.ExecutableLinux=/usr/bin/xvfb-run
+App.WorkingDir=1430110/server
+App.LinuxCommandLineArgs=-a wine "./accServer.exe"
+App.WindowsCommandLineArgs=
+App.CommandLineArgs={{$PlatformArgs}} {{$FormattedArgs}}
+App.AppSettings={}
+App.EnvironmentVariables={"LD_LIBRARY_PATH":"{{$FullBaseDir}}linux64:%LD_LIBRARY_PATH%","SteamAppId":"805550"}
+App.CommandLineParameterFormat=-{0} "{1}"
+App.CommandLineParameterDelimiter= 
+App.ExitMethod=OS_CLOSE
+App.ExitTimeout=30
+App.ExitString=stop
+App.ExitFile=app_exit.lck
+App.HasWriteableConsole=True
+App.HasReadableConsole=True
+App.SupportsLiveSettingsChanges=False
+App.LiveSettingChangeCommandFormat=set {0} "{1}"
+App.ApplicationIPBinding=0.0.0.0
+App.Ports=[{"Protocol":"UDP","Port":9231,"Ref":"UDPPort","Name":"UDP Port","Description":"Port for game traffic"},{"Protocol":"TCP","Port":9232,"Ref":"TCPPort","Name":"TCP Port","Description":"Port for server query and ping"}]
+App.PrimaryApplicationPortRef=UDPPort
+App.UniversalSleepApplicationUDPPortRef=UDPPort
+App.UniversalSleepSteamQueryPortRef=TCPPort
+App.MaxUsers=85
+App.UseRandomAdminPassword=False
+App.RemoteAdminPassword=Password123
+App.AdminMethod=STDIO
+App.AdminLoginTransform=None
+App.RCONConnectDelaySeconds=5
+App.RCONConnectRetrySeconds=5
+App.TelnetLoginFormat={0}
+App.UpdateSources=[{"UpdateStageName":"SteamCMD Server Download","UpdateSourcePlatform":"All","UpdateSource":"SteamCMD","UpdateSourceData":"1430110","ForceDownloadPlatform":"Windows"}]
+App.Compatibility=LinuxBufferedConsole, WindowsBufferedConsole
+App.SteamUpdateAnonymousLogin=False
+App.SteamForceLoginPrompt=False
+App.RapidStartup=False
+App.MonitorChildProcess=True
+App.MonitorChildProcessWaitMs=2000
+App.MonitorChildProcessName=wine64
+App.SupportsUniversalSleep=False
+App.WakeupMode=Any
+App.ApplicationReadyMode=RegexMatch
+App.TemplateMatchRegex={{(\$?[\w]+)}}
+Console.FilterMatchRegex=
+Console.FilterMatchReplacement=
+Console.ThrowawayMessageRegex=^((WARNING|ERROR): Shader.+|\d+:err:ole:.*|Could not find Wine Gecko.*)$
+Console.AppReadyRegex=^Listening to TCP \d+ \| UDP \d+$
+Console.UserJoinRegex=^New connection request: id (?<sessionid>.+?) (?<username>.+?) (?<userid>.+?) on car model \d+$
+Console.UserLeaveRegex=^Client (?<sessionid>.+?) closed the connection \(\d+\)$
+Console.UserChatRegex=^CHAT (?<username>.+?): (?<message>.+)$
```

Comment:
Remove the regex since there's none for this game that will work in AMP. Join and chat, but no leave.

Reply:
Hmmm. Pretty sure I got that regex from someone else who tested the template. Assume you tried on Windows?

Reply:
I'll share the log in Discord.

Reply:
Confirmed you have it correct. I missed the line that had the connection close.